artemis-ra/src/main/java/org/apache/activemq/artemis/ra/ActiveMQResourceAdapter.java

```diff
@@ -821,6 +821,10 @@ public void setUseTopologyForLoadBalancing(Boolean useTopologyForLoadBalancing)
       raProperties.setUseTopologyForLoadBalancing(useTopologyForLoadBalancing);
    }
 
+   public Boolean getUseTopologyForLoadBalancing() {
+      return raProperties.isUseTopologyForLoadBalancing();
```

Comment:
why?


isUseTopology is the standard for Java Beans.

Why do you need the get method?

Reply:
Unfortunately, this is not recognized in some environments.  Websphere Application Server 8.x is one of these.